share/runtime-install.tmpl

```diff
@@ -91,14 +86,14 @@ installpkg device-mapper-persistent-data
 installpkg xfsdump
 
 ## needed for LUKS escrow
-installpkg python-volume_key volume_key
-installpkg python-nss
+installpkg volume_key
+installpkg nss-tools
 
 ## SELinux support
-installpkg selinux-policy-targeted audit libsemanage-python
+installpkg selinux-policy-targeted audit
 
 ## network tools/servers
-installpkg python-ethtool ethtool openssh-server nfs-utils openssh-clients
+installpkg ethtool openssh-server nfs-utils openssh-clients
```

Comment:
Do we still need ``ethtool``? I remember somebody mentioning it recently in relation to some deprecated/removed boot option or whatever.